ui/src/writeData/constants/contentClientLibraries.ts

```diff
@@ -0,0 +1,112 @@
+// Constants
+import {CLIENT_LIBS} from 'src/shared/constants/routes'
+
+// Types
+import {WriteDataItem, WriteDataSection} from 'src/writeData/constants'
+
+// Markdown
+import ArduinoMarkdown from 'src/writeData/components/clientLibraries/Arduino.md'
```

Comment:
if only there was a way to self register these to the page.... where have i seen that....